### PR TITLE
chore(orc8r): Add API for SGi IPv6 address

### PR DESCRIPTION
**Pull request by pshelar**
This would allow operator set IPv6 address on SGi interface.
Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by koolzz — APPROVED** [review]

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go [hunk 1]
```diff
@@ -545,7 +545,7 @@ func TestBuilder_Build_NonNat(t *testing.T) {
 	// validate SGi vlan tag mconfig
 	lteGW = configurator.NetworkEntity{
 		Type: lte.CellularGatewayEntityType, Key: "gw1",
-		Config:             newGatewayConfigNonNat("55", "1.2.3.4/24", "1.2.3.1"),
+		Config:             newGatewayConfigNonNat("55", "1.2.3.4/24", "1.2.3.1", "", ""),
```
Comment (hcgatewood) [inline lte/cloud/go/services/lte/servicers/builder_servicer_test.go:548]: Nit: these types of multi-arg same-type functions are bug-adjacent -- preference is to pass in a typed struct object as the single arg

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go [hunk 2]
```diff
@@ -1570,7 +1641,8 @@ func newDefaultGatewayConfig() *lte_models.GatewayCellularConfigs {
 	}
 }
 
-func newGatewayConfigNonNat(vlan string, sgi_ip string, sgi_gw string) *lte_models.GatewayCellularConfigs {
+func newGatewayConfigNonNat(vlan string, sgi_ip string, sgi_gw string,
```
Comment (hcgatewood) [inline lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: Nit: move the args each to their own line

**Review by hcgatewood — APPROVED** [review]
Lgtm! Couple optional nits

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go (hunk 1 above)
Comment (pshelar) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go:548]: I will add todo in doe this.

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go (hunk 2 above)
Comment (pshelar) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: following existing coding style.

File: lte/protos/mconfig/mconfigs.proto [hunk 3]
```diff
@@ -175,6 +175,9 @@ message PipelineD {
     bool enable5g_features = 18;
     string upf_node_identifier = 19;
 
+    string sgi_management_iface_ipv6_addr = 20;
```
Comment (andreilee) [inline lte/protos/mconfig/mconfigs.proto:179]: Could you add a description to these new fields, similar to what's been added on the swagger files

**Review by andreilee — APPROVED** [review]

File: lte/protos/mconfig/mconfigs.proto (hunk 3 above)
Comment (pshelar) [reply lte/protos/mconfig/mconfigs.proto:179]: done

**Pull request merged**